Added Code_Of_Conduct.md

ShivamVerma5665 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.

You can learn more about contributing to Sketchify here: https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md

Happy contributing!

-->

## Link to the Issue
#13 

## Description of your PR
Added CODE_OF_CONDUCT.md 

## Screenshots (if applicable)
-

## Test Plan
I have added the code of conduct document which clarifies sketchify's mission, values and principles, linking them with standards of professional conduct.

### Have you read the [Contributing Guidelines](https://github.com/s-katte/React-Code-Pen/blob/master/CONTRIBUTING.md)?
Yes, I have.

Commit:
Added Code_Of_Conduct

s-katte reviewed (changes requested):
content looks fine, few changes here:

- Headings are not reflected (add space between '#' and words)

ShivamVerma5665 commented:
> content looks fine, few changes here:
> 
>     * Headings are not reflected (add space between '#' and words)
Okay, will do it.

Commit:
Update CODE_OF_CONDUCT.md

As requested, the changes have been made related to he headings of this document.

s-katte reviewed (approved):
Looks good to me.
Merging it.
